docs/one-click-integrations/jira-integration.md

```diff
@@ -15,6 +15,9 @@ To set up the Jira integration:
 
     ![Installing the Pulse Jira app](images/jira-installing.png)
 
+    !!! warning
+        It is not possible to install Pulse's Jira application twice on the same Jira instance, using two different Pulse organizations.
```

Comment:
The following edit is to map the statement better to the use case of the users: users want to integrate multiple Pulse organizations with the same Jira instance and aren't necessarily concerned if they can install the Pulse Jira application multiple times or not. The edit also makes the sentence sound more "positive" without losing clarity.

```suggestion
        You can only integrate one Pulse organization with each Jira instance.
```